learning/katas/go/Core Transforms/Map/ParDo OneToMany/task.md

```diff
@@ -26,7 +26,7 @@ In this kata we will map a single element into many by splitting a sentence into
 
 <div class="hint">
   Use <a href="https://godoc.org/github.com/apache/beam/sdks/go/pkg/beam#ParDo">
-  ParDo</a>
+  beam.ParDo</a>
   with a <a href="https://godoc.org/github.com/apache/beam/sdks/go/pkg/beam#hdr-DoFns">
   DoFn</a>.
```

Comment:
Should this be `beam.DoFn` too?

Reply:
Thank you, Henry, for reviewing.  I left out the `beam.` purposely since we don't import a DoFn from the beam package but provide it to our `beam.ParDo` as a `func` or `struct` in our codebase.

Reply:
+1 to Damon's response. There's no such thing as beam.DoFn as an entity in code.